saas/management/commands/report_weekly_revenue.py

```diff
@@ -73,30 +73,30 @@ def construct_date_periods(self, today, timezone=None):
 
     def construct_table(self, data):
         table = {
-            'total_sales': {
+            'Total Sales': {
                 'last': data['account_table'][0]['values'][1][1],
```

Comment:
Those values are in cents because we store the amounts as integers in the database. It needs to be converted with `as_money` in `saas.humanize`.
Also percentages must be formatted with only 2 decimals after the coma.